docs/ABI/CallConvSummary.rst

```diff
@@ -0,0 +1,188 @@
+| ``rbp``   | Callee-saved register,           |          |          |          |
+|           | used as frame pointer            |          |          |          |
++-----------+----------------------------------+----------+----------+----------+
+
+Stack frame
+^^^^^^^^^^^
+
+On function entry, ``sp+8`` is **16-byte aligned**, i.e. the start of the memory
+arguments is 16-byte aligned.
+
++---------------+---------------+------------------------+
+|               | ``rbp+8n+16`` | memory argument *n*    |
+|               |               |                        |
+|               | ...           | ...                    |
+|               |               |                        |
+|               | ``rbp+16``    | memory argument 0      |
++---------------+-----------+---+------------------------+
+| ↓ Current Frame           |           ↑ Previous Frame |
++---------------+-----------+---+------------------------+
+|               | ``rbp+8``     | return address         |
+|               |               |                        |
++---------------+---------------+------------------------+
+| entry ``rsp`` | ``rbp``       | previous ``rbp`` value |
++---------------+---------------+------------------------+
+|               | ``rbp-8``     |                        |
+|               |               |                        |
+|               | ...           |      local storage     |
+|               |               |                        |
+|               | ``rsp``       |                        |
++---------------+---------------+------------------------+
+|               | ``rsp-8``     |                        |
+|               |               |                        |
+|               | ...           |        red zone        |
+|               |               |                        |
+|               | ``rsp-128``   |                        |
++---------------+---------------+------------------------+
+
+ARM64
+-----
+
+See `Apple ARM64 Documentation`_, `Procedure Call Standard for the Arm 64-bit Architecture`_.
+
+.. _Apple ARM64 Documentation: https://developer.apple.com/documentation/xcode/writing-arm64-code-for-apple-platforms
+.. _Procedure Call Standard for the Arm 64-bit Architecture: https://github.com/ARM-software/abi-aa/blob/main/aapcs64/aapcs64.rst
+
+Register usage
+^^^^^^^^^^^^^^
+
++----------+---------+-------------------------+----------+----------+----------+
+| Register | Special | Purpose                 | C++      | ObjC     | Swift    |
++==========+=========+=========================+==========+==========+==========+
+| ``x0``   |         | Integer argument 1      | ``this`` | ``self`` |          |
+|          |         | (1st return value)      |          |          |          |
++----------+---------+-------------------------+----------+----------+----------+
+| ``x1``   |         | Integer argument 2      | ``_cmd`` |          |          |
```

Comment:
`_cmd` should go in the ObjC column, I believe.

Reply:
Oops. Yes. It was in the original before I converted it to RST. Well spotted. :-)